### PR TITLE
Playground Section

### PR DESCRIPTION
**Pull request by Gaisensai**
- Fixed button background color on homepage
- Created Playground page, Key Extraction Page, Summarization api page
Screenshot (29)
Screenshot (27)
Screenshot (28)

### CODE REVIEW

**Comment by PhilemonBrain** [comment]
@Gaisensai thanks a lot for this work. Please include the playground section on the navbar

**Review by Vinyl-Davyl — APPROVED** [review]

**Comment by Vinyl-Davyl** [comment]
@Gaisensai Nicee. there is no route to the playgroud section and no redirect link from any of the buttons, playground also needs to be on the navbar as it is in the design.

**Review by Vinyl-Davyl — DISMISSED** [review]
@Gaisensai Nicee. there is no route to the playgroud section and no redirect link from any of the buttons, playground also needs to be on the navbar as it is in the design.

File: src/assets/icons/index.ts
Comment (pablo-clueless) [inline src/assets/icons/index.ts:1]: By the time this file is gzipped, it's gonna be larger than 500kb. You should created a new file for the playground icon.

Comment (Gaisensai) [reply src/assets/icons/index.ts:1]: I'll do that

**Comment by Gaisensai** [comment]
Screenshot (30)

**Pull request merged**